src/Forms/HTMLEditor/HTMLEditorField.php

```diff
@@ -185,4 +185,16 @@ public function Field($properties = array())
         $this->getEditorConfig()->init();
         return parent::Field($properties);
     }
+
+    public function getSchemaStateDefaults()
+    {
+        $stateDefaults = parent::getSchemaStateDefaults();
+        $config = $this->getEditorConfig();
+        $stateDefaults['data']['config'] = $config->getAttributes();
```

Comment:
maybe this should be named more appropriately as `$stateDefaults['data']['attributes']`, looking from the otherside it's quite confusing

Reply:
Seems fair :)